### PR TITLE
Added reason to response body

### PR DESCRIPTION
**Pull request by Anlord241**

### CODE REVIEW

**Review by Darina007 — APPROVED** [review]

**Review by AntonMaksimenkau — CHANGES_REQUESTED** [review]
Pretty brave deceison. It can compromise some data.
Let's find another way

**Review by maxim12321 — CHANGES_REQUESTED** [review]
Let's use one ControllerAdvice class for all exception handlers

**Review by AntonMaksimenkau — APPROVED** [review]

File: src/main/java/com/team4/testingsystem/exceptions/Response.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.team4.testingsystem.exceptions;
```
Comment (AntonMaksimenkau) [inline src/main/java/com/team4/testingsystem/exceptions/Response.java]: Response doesn't seam like an exception. Let's move it somewhere

File: src/main/java/com/team4/testingsystem/exceptions/ControllerAdvice.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+public class ControllerAdvice extends ResponseEntityExceptionHandler {
+
+    @ExceptionHandler(value = {ConflictException.class})
+    @ResponseStatus(value = HttpStatus.CONFLICT)
+    public ErrorResponse handleConflictException(ConflictException e) {
+        return new ErrorResponse(e.getMessage());
+    }
+
+    @ExceptionHandler(value = {FileOperationException.class})
+    @ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
```
Comment (maxim12321) [inline src/main/java/com/team4/testingsystem/exceptions/ControllerAdvice.java]: Maybe we should send 4xx responses only?

Comment (Anlord241) [reply src/main/java/com/team4/testingsystem/exceptions/ControllerAdvice.java]: What code should I use then?

Comment (Anlord241) [reply src/main/java/com/team4/testingsystem/exceptions/ControllerAdvice.java]: I think problems with file saving/deleting are server errors but not client

Comment (maxim12321) [reply src/main/java/com/team4/testingsystem/exceptions/ControllerAdvice.java]: `409 Conflict`? `417 Expectation Failed`?
`418 I'm a teapot` :)

**Review by maxim12321 — APPROVED** [review]

**Pull request merged**